application-portal/src/app/models/choices/choices.ts

```diff
@@ -1,7 +1,10 @@
 export class Choices {
```

Comment:
Model names should be singular, i.e. this should be `Choice` or better `ProgramChoice`